### DISCUSSION TITLE
MMKV Persister?

### DISCUSSION THREAD

**Discussion by rikur** [tinyplex/tinybase#129] [Ideas]
Performance seems great:
https://github.com/Tencent/MMKV/wiki/android_benchmark
https://github.com/Tencent/MMKV/wiki/iOS_benchmark

**Comment by jamesgpearce** [comment]
Interesting! What's the story for integrating it into a JavaScript environment?

**Reply by waynesbrain** [reply]
There's a binding for React Native here https://github.com/mrousavy/react-native-mmkv
